Source/Plugins/DomainRandomizationDNN/Source/DomainRandomizationDNN/Public/GroupActorManager.cpp

```diff
@@ -382,9 +382,9 @@ void AGroupActorManager::UpdateProxyMeshes()
             NewProxyMeshComponent->bIsEditorOnly = true;
             NewProxyMeshComponent->SetCollisionProfileName(UCollisionProfile::NoCollision_ProfileName);
             NewProxyMeshComponent->bHiddenInGame = true;
-            NewProxyMeshComponent->bVisible = !bShouldHideProxyMesh;
+            NewProxyMeshComponent->SetVisibility(!bShouldHideProxyMesh);
             NewProxyMeshComponent->CastShadow = false;
-            NewProxyMeshComponent->PostPhysicsComponentTick.bCanEverTick = false;
+            //NewProxyMeshComponent->PostPhysicsComponentTick.bCanEverTick = false;
```

Comment:
Why do we need to comment out this line?

Reply:
PostPhysicsComponentTick is not a member of that class anymore. i can't say the implication of not setting the tick to false. so i prefered to comment it out